assembly/assembly-wsmaster-war/src/main/webapp/WEB-INF/classes/che/che.properties

```diff
@@ -196,6 +188,18 @@ che.oauth.openshift.verify_token_url= NULL
 # a recurring schedule.
 schedule.core_pool_size=10
 
+# Name of cofig map in Che server namespace with additional CA TLS certificates to be propogated into all user's workspaces.
+# If the property is set on OpenShift 4 infrastructure, then cluster CA bundle will be propogated too.
+che.trusted_ca.bundles_configmap=NULL
+
+# Name of configmap in a workspace namespace with additional CA TLS certificates.
+# Holds the copy of che.trusted_ca.bundles_configmap but in a workspace namespace.
+# Content of this config map is mounted into all workspace containers including plugin brokers.
+che.trusted_ca.workspace_bundle_configmap=ca-certs
+
+# Configures path on workspace containers where the CA bundles are mount.
+che.trusted_ca.bundle_mount_path=/public-certs
```

Comment:
Since the doc says `path on workspace containers`, is it a good idea to add workspace word in property name?

Reply:
This property specifies the path on which CA bundle config map should be mounted inside container. This is done for all workspace containers as well as plugin brokers. I may add the word `workspace` if you think it is useful there.